### DISCUSSION TITLE
How to import a node package (as plugin)?

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#19062] [Questions]
I am trying to add some client side libraries.

In Nuxt 2 I have worked with this pattern:

```js
export default defineNuxtConfig({
    plugins: [
        { src: "~/node_modules/focus-visible", mode: "client" },
    ],
});
```
(without the defineNuxtConfig of course)

In Nuxt 3 the above code does not work. It produces an error: `Uncaught SyntaxError: ambiguous indirect export: default`

Is it a bug?
How to simply import packages/scripts?

**Comment by TheAlexLichter** [comment]
No, this is no bug. 
What you want it creating a new file in the `plugins` folder and import the package there.

**Reply by ghost** [reply]
I’ve tried this, but I get the same error …

**Reply by TheAlexLichter** [reply]
Then feel free to share a reproduction ☺️
